config/cmake/buildSundials.cmake

```diff
@@ -43,41 +43,42 @@ function (build_sundials_msvc)
         set(SUNDIALS_KLU_ENABLE "-DKLU_ENABLE=OFF")
     endif()
     set(CMAKE_LIST_CONTENT "
-    cmake_minimum_required(VERSION 3.5)
-    include(ExternalProject)
-ExternalProject_Add(sundials
-    SOURCE_DIR ${PROJECT_BINARY_DIR}/Download/sundials
-    GIT_REPOSITORY  https://github.com/llnl/sundials.git
-    GIT_TAG v3.1.1
-    UPDATE_COMMAND " "
-    BINARY_DIR ${PROJECT_BINARY_DIR}/ThirdParty/sundials
+        cmake_minimum_required(VERSION 3.5)
```

Comment:
Is this just a reformat?  or was something else changed

Reply:
Should just be a reformat